source/Halibut.Tests/Support/TestAttributes/SyncAndAsyncAttribute.cs

```diff
@@ -93,6 +96,18 @@ public static AsyncHalibutFeature ToAsyncHalibutFeature(this SyncOrAsync syncOrA
                 _ => throw new ArgumentOutOfRangeException(nameof(syncOrAsync), syncOrAsync, null)
             };
         }
+
+        public static async Task WriteToStream(this SyncOrAsync syncOrAsync, Stream stream, byte[] bytes)
```

Comment:
Do these make it worse or better?

Reply:
If it was used in multiple test classes, I'd say better for sure.

For now, it's only used in 1, so the benefit isn't really there. But if we foresee more usages of this (as streams are common), this can turn into useful easily.